packages/slice-machine/src/modules/slices/index.ts

```diff
@@ -67,7 +68,12 @@ export function* createSliceSaga({
     payload.sliceName,
     payload.libName
   )) as SagaReturnType<typeof createSlice>;
+  void Tracker.get().trackCreateSlice({
+    id: payload.sliceName,
+    name: payload.libName,
```

Comment:
are you sure that the properties are right ?

`LibName` doesn't sound like a `SliceName`
and
`SliceName` doesn't sound like `sliceId`.

Reply:
Those are the only two properties given in the request when creating a slice,